frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -213,5 +222,8 @@ export const detectGitRepoName = (url: string): string | undefined => {
     return undefined;
   }
 
-  return url.split('/').pop();
+  return url
+    .split('/')
+    .pop()
+    .replace('.git', '');
```

Comment:
@invincibleJai Why not replace `.` with `-`? That will make it even work with names with `.` in between.

Reply:
@rohitkrai03 Done